docs/UserGuide.md

```diff
@@ -346,25 +346,26 @@ Therefore, to change the status of a leave, please use the <a href= "#approve-le
 
 ### Removing a leave record: `delete-leave`
 
-* **What It Does:**
-  * Removes an existing leave record form HRMate, so that the leave's record is no longer available in HRMate.
-* **Format:**
-  * `delete-leave LEAVE_LIST_INDEX`
-* **Examples:**
 
-| S/N | Command information                                                                                                                                                                                                                                                    |
-|-----|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| 1   | **Command:** `delete-leave 1` <br><br> **Output:** Deletes the leave record with index 1. For instance, if the leave record titled "medical leave" with employee "David de Gea" is indexed 1, then "medical leave" is removed from HRMate and no longer available.     |
-| 2   | **Command:** `delete-leave 2` <br><br> **Output:** Deletes the leave record with index 2. For instance, if the leave record titled "childcare leave" with employee "Carlos Puyol" is indexed 2, then "childcare leave" is removed from HRMate and no longer available. |
+If you wish to remove a leave record from HRMate, you can use the `delete-leave` command to do so.
+
+
+1. Type in the following [command](#glossary) in the [command box](#glossary): `delete-leave LEAVE_LIST_INDEX`, and replace `LEAVE_LIST_INDEX` with the index of the leave record you wish to remove.
+ - For instance: if you wanted to remove the leave record titled "medical leave" with employee "David de Gea", you would type in `delete-leave 1` in the command box.
```

Comment:
might be good to add a screenshot for this, the index might be confusing for people